vagrant/provisioning/setup-k8s-minion.sh

```diff
@@ -11,6 +11,10 @@ MASTER_IP=$1
 
 echo "MASTER_IP=$MASTER_IP" >> setup_minion_args.sh
 
+# ensure things that ovnkube depends on are started
```

Comment:
You don't need these 2 commands. In debian, when packages are installed, the services automatically start.

Reply:
@shettyg I noticed that even nodes have northd running, probably because setup-minion.sh installs openvswitch-ovn-central.  Do we really need that for nodes?

Reply:
@dcbw That was an oversight. No need for central in minions.